### PR TITLE
numaresources-operator: 4.10 branching update 

### PR DESCRIPTION
**Pull request by marioferh**
Create 4.10 config file
Update quay mirroring

Signed-off-by: Mario Fernandez <mariofer@redhat.com>

### CODE REVIEW

**Comment by Tal-or** [comment]
There should be another prow config file for 4.10 like we have here:
https://github.com/openshift/release/tree/master/ci-operator/config/openshift-kni/scheduler-plugins

**Comment by marioferh** [comment]
> There should be another prow config file for 4.10 like we have here: https://github.com/openshift/release/tree/master/ci-operator/config/openshift-kni/scheduler-plugins

I've missed to upload the file sorry.

File: core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # numaresources-operator
 registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator quay.io/openshift-kni/numaresources-operator:4.10.999-snapshot
```
[the remark this answers is not in the document]
Comment (marioferh) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:2]: There is no 4.11.999-snapshot here https://quay.io/repository/openshift-kni/numaresources-operator?tab=tags

[the remark this answers is not in the document]
Comment (Tal-or) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:2]: Right. once you'll update here, we'll have :)

[the remark this answers is not in the document]
Comment (marioferh) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:2]: ahh got it, thanks

File: ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml [hunk 2]
```diff
@@ -38,12 +38,12 @@ promotion:
 releases:
   initial:
     integration:
-      name: "4.10"
+      name: "4.11"
```
Comment (fromanirh) [inline ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:41]: does the bundle pullspec (line 33 above) need a similar replacement?

**Comment by fromanirh** [comment]
> [APPROVALNOTIFIER] This PR is **NOT APPROVED**
> 
> This pull-request has been approved by: _[marioferh](https://github.com/openshift/release/pull/26142#)_ To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **cynepco3hahue** and additionally assign **cgwalters**, **eranco74**, **hardys**, **sjenning**, **wking** after the PR has been reviewed. You can assign the PR to them by writing `/assign @cgwalters @eranco74 @hardys @sjenning @wking` in a comment when ready.
> 
> The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=openshift%2Frelease).
> Needs approval from an approver in each of these files:
> 
>     * **[ci-operator/config/openshift-kni/numaresources-operator/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/numaresources-operator/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift-kni/numaresources-operator/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift-kni/numaresources-operator/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift-priv/installer/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift-priv/installer/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift-priv/machine-config-operator/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift-priv/machine-config-operator/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/assisted-installer/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/assisted-installer/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/assisted-installer-agent/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/assisted-installer-agent/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/assisted-service/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/assisted-service/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/assisted-test-infra/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/assisted-test-infra/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/cluster-api-provider-agent/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/cluster-api-provider-agent/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/cluster-baremetal-operator/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/cluster-baremetal-operator/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/hypershift/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/hypershift/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/installer/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/installer/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/machine-config-operator/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/machine-config-operator/OWNERS)**
> 
>     * **[ci-operator/jobs/openshift/release/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/OWNERS)**
> 
>     * **[core-services/image-mirroring/openshift-kni/OWNERS](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift-kni/OWNERS)**
> 
> 
> Approvers can indicate their approval by writing `/approve` in a comment Approvers can cancel approval by writing `/approve cancel` in a comment

this cannot be right. Unrelated changes have sneaked inside this PR.

File: ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml [hunk 3]
```diff
@@ -178,7 +178,7 @@ presubmits:
       skip_cloning: true
     hidden: true
     labels:
-      ci-operator.openshift.io/cloud: packet
+      ci-operator.openshift.io/cloud: packet-edge
```
[the remark this answers is not in the document]
Comment (marioferh) [reply ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml:181]: These changes comes from:
make ci-operator-config
make jobs
Investigating...

File: ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml (hunk 2 above)
Comment (marioferh) [reply ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:41]: I think so

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**